GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Employer.cshtml

```diff
@@ -116,10 +117,11 @@
                                         {
                                             if (organisation.GetIsInscope(report.AccountingDate.Year))
                                             {
+                                                DateTime deadline = ReportingYearsHelper.GetDeadlineForAccountingDate(report.AccountingDate);
                                                 <span>
                                                     This organisation

```

Comment:
It's probably covered by the other content tickets (the 6 I'm halfway through) but we should be changing organisation to employer. This is perhaps not the best ticket to do it on though, might as well group it with the rest of those changes